db/migrate/20211214092141_add_reference_for_raised_indicators_and_voting_indicators.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class AddReferenceForRaisedIndicatorsAndVotingIndicators < ActiveRecord::Migration[6.1]
+  def up
+    add_column :raised_indicators, :uuid, :uuid, default: "gen_random_uuid()", null: false
+    rename_column :raised_indicators, :id, :integer_id
```

Comment:
@hengsokly what is the purpose of this rename_column?

Reply:
I want to use column id(data type as UUID) as the primary column